src/main/java/org/tikv/txn/TwoPhaseCommitter.java

```diff
@@ -101,6 +101,21 @@ public TwoPhaseCommitter(TiSession session, long startTime, long lockTTL) {
         createExecutorService(WRITE_BUFFER_SIZE));
   }
 
+  public TwoPhaseCommitter(
```

Comment:
change previous constructor method to `this(session,startTime,lockTTL,createExecutorService(WRITE_BUFFER_SIZE));`
![image](https://github.com/tikv/client-java/assets/52435083/29cb841c-153f-4af1-ae6d-ed23a335c94b)